products/tests.py

```diff
@@ -44,4 +49,77 @@ def test_listcategory_get_error(self):
             )
             self.assertEqual(response.status_code, 404)
 
+class LikeTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        User.objects.bulk_create([
+            User(id=1,kakao_id=1, nickname="palli"),
+            User(id=2, kakao_id=2, nickname="dong")
```

Comment:
- 코드를 작성하실때는 항상 팀에 몇명이 있던 한 사람이 작업한 것처럼 깔끔하게 작성해주시는게 규모가 커질수록 중요해집니다.
- 전체적으로 `,` 앞뒤 간격이 통일되지 않은 것 같은데 아래와 같이 수정해주세요.
```suggestion
            User(id=1, kakao_id=1, nickname="palli"),
            User(id=2, kakao_id=2, nickname="dong")
```

Reply:
반영했습니다!